tools/noise/subr.sh

```diff
@@ -299,6 +299,9 @@ get_image_iso() {
 	if [ -z "$iso" -o "$iso" = "65535" ]; then
 		iso=$(get_exif_key "$file" Exif.Photo.StandardOutputSensitivity)
 	fi
+	if [ -z "$iso" -o "$iso" = "65535" ]; then
```

Comment:
it looks like the dt proper uses following tags:
    "Exif.Photo.ISOSpeedRatings", "Exif.Photo.RecommendedExposureIndex",
    "Exif.Photo.StandardOutputSensitivity", "Exif.Nikon3.ISOSpeed",
    "Exif.Nikon3.ISOSettings", "Exif.NikonIi.ISO"
https://github.com/darktable-org/darktable/blob/62416056d956a05f891846dc78fc11033d3c699b/tools/dngmeta.rb#L40-L42

I wonder if this also should read them all